api/src/main/java/org/openmrs/module/fhir/api/strategies/relatedperson/RelatedPersonStrategy.java

```diff
@@ -27,6 +29,31 @@ public RelatedPerson getRelatedPerson(String uuid) {
 		return FHIRRelatedPersonUtil.generateRelationshipObject(omrsRelationship);
 	}
 
+	/**
+	 * @see org.openmrs.module.fhir.api.strategies.relatedperson.GenericRelatedPersonStrategy#searchByPatientIdentifier(String)
+	 */
+	@Override
+	public List<RelatedPerson> searchByPatientIdentifier(String identifierValue) {
+		List<RelatedPerson> relatedPersonList = new ArrayList<>();
+		List<Relationship> relationships = new ArrayList<>();
+		List<PatientIdentifierType> allPatientIdentifierTypes = Context.getPatientService().getAllPatientIdentifierTypes();
+		List<org.openmrs.Patient> patientList = Context.getPatientService().getPatients(identifierValue,
+				null, allPatientIdentifierTypes, true);
+
+		if (patientList != null && !patientList.isEmpty()) {
+			for (Patient patient : patientList) {
+				relationships.addAll(Context.getPersonService().getRelationshipsByPerson(patient));
```

Comment:
Instead of calling Context.getPersonService() again for each item in the list, why not do it once?